guides/tfvc-to-git.md

```diff
@@ -1,47 +1,220 @@
-# TFVC (Team Foundation Version Control) to Git
-
-**Similar Guides:**  
+<!-- markdownlint-disable MD024 MD031 MD032 -->
+<!-- MD024 - Multiple headings with the same content
+     Disabled for the TFS, GCcode and GitHub sub-sections. No other headings
+     make sense.-->
+<!-- MD031 - Fenced code blocks should be surrounded by blank lines
+     Disabled for rendering and formatting purposes. With blank lines, the
+     numbered lists are broken when rendered by the Architect GitHub Pages 
+     theme. -->
+<!-- MD032 - Lists should be surrounded by blank lines
+     Disabled for rendering and formatting purposes. With blank lines, the
+     numbered lists are broken when rendered by the Architect GitHub Pages 
+     theme. -->
+
+# Migrate from TFVC to Git in TFS, GCcode or GitHub
```

Comment:
Probably more of a personal preference, the title shouldn't reference the platform for the migration (GCcode/TFS). Having the background does the job much better.

Reply:
Yes, I agree. I originally wanted to drive in the fact that people can still stick with TFS if they wanted to, but I have that in the `Background` section now. I'll remove it from the title.

Reply:
Resolved with latest push.